.github/workflows/ci-cd.yml

```diff
@@ -451,7 +451,7 @@ jobs:
 
   CI-status-report:
     runs-on: ubuntu-latest
-    needs: [rspec, jest, cypress, build_test]
+    needs: [rspec, jest, cypress, build_test, audit, storybook]
```

Comment:
CI status should also be making sure audit and storybook passed. This oversight made merge a broken dependency update by accident.